controllers/pullRequestsController.js

```diff
@@ -0,0 +1,35 @@
+const logger = require('../utils/logger')
+const fetch = require('../lib/fetch')
+const getGithubId = require('../utils/getGithubId')
+
+const pullRequests = async (req, res) => {
+  try {
+    const githubId = await getGithubId(req.params.id)
+    const url = `https://api.github.com/search/issues?q=org:Real-Dev-Squad+author:${githubId}+type:pr`
```

Comment:
Add the github API URL in the config: https://github.com/Real-Dev-Squad/website-backend/blob/develop/config/default.js#L18

The org name also should be added in the config.